zk-sdk/src/encryption/elgamal.rs

```diff
@@ -719,9 +723,13 @@ impl DecryptHandle {
             return None;
         }
 
-        Some(DecryptHandle(
-            CompressedRistretto::from_slice(bytes).decompress()?,
-        ))
+        let Ok(compressed_ristretto) = CompressedRistretto::from_slice(bytes) else {
+            return None;
+        };
+
+        let ristretto_point = compressed_ristretto.decompress()?;
+
+        Some(DecryptHandle(ristretto_point))
```

Comment:
Same here, and else where - `ok_or()` and other helper functions are fine when used sensibly.
People who write Rust should be able to read the following with no problem, it is a very common pattern:

```suggestion
        let Ok(compressed_ristretto) = CompressedRistretto::from_slice(bytes) else {
            return None;
        };
        compressed_ristretto.decompress()
            .map(DecryptHandle)
```